LanguageFeatures/Wildcards/initializing_formals_A02_t02.dart

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion No parameter with the name `_` is bound, which means `_` can't be
+/// accessed inside the initializer list. The name `_` can be used in the body,
+/// but this is a reference to the field, not to the parameter
+///
+/// @description Checks that a class member named `_` can be initialized in the
+/// initializer list.
+/// @author sgrekhov22@gmail.com
+
+// SharedOptions=--enable-experiment=wildcard-variables
+
+import '../../Utils/expect.dart';
+
+class C1 {
+  int _, a;
+  C1(this.a) : _ = a;
+}
+
+class C2 {
+  final int _, a;
+  const C2(this.a) : _ = a;
```

Comment:
How about this case, is that already covered?:
```suggestion
  const C2(this.a) : _ = a;
  const C2(this.a, bool _) : _ = a; // OK, there is no name clash with the wildcard parameter.
```

Reply:
Added

Reply:
Thx!